### ISSUE TITLE
Long pressing back button history and inconsistent edge swipe back and forward gestures in tab

### ISSUE THREAD

**Issue by BugReportI** [uazo/bromite-buildtools#292]
Is it possible to add long pressing back key in navigation bar to open tab history?

I know Firefox has this feature and Bromite use to have it but got removed at some point.

The edge gestures used to go back and forward are inconsistent on websites. I can sometimes use it and sometimes it just will not work.

**Comment by uazo** [comment]
> Is it possible to add long pressing back key in navigation bar to open tab history?

Is already so, never removed.

> The edge gestures used to go back and forward are inconsistent on websites. I can sometimes use it and sometimes it just will not work.

unfortunately we don't have a parallel chromium version to try, can you check with chrome?

**Comment by BugReportI** [comment]
> Is already so, never removed.
> 
> unfortunately we don't have a parallel chromium version to try, can you check with chrome?

Whenever I press back in navigation bar it just takes me back to previous page.

I will try Chromium version and let you know the results.

**Comment by uazo** [comment]
>> Is already so, never removed.
>
> Whenever I press back in navigation bar it just takes me back to previous page.

Sorry, back? I mean long press back.

**Comment by BugReportI** [comment]
> Sorry, back? I mean long press back.

I  am talking about back key in three key navigation bar. Long pressing this key just takes me back to previous page.

If you mean long pressing the back and forward buttons in menu than it does not work there either.

**Comment by BugReportI** [comment]
I tried this back and forward edge gestures navigation to go previous and forward pages in Microsoft Edge and Google Chrome and it does not work In either of them.

I also tried to access tab history and it works in Microsoft Edge by pressing the back or forward buttons, showing respectively previous and forward tab history.

I cannot do access tab history through these buttons in Google Chrome and I cannot access by pressing back key in three key navigation in both Microsoft Edge and Google Chrome.
